hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(0xff));
```

Comment:
This is the first use of the testbi.  Part of this issue is mapping the semantics
of the Vinstrs which were based off of x86 onto ARM.    You're right.  It's ignoring
s0.  

Previously, the assembler would eventually recognize the tst alias.  I thought the
previous comment was to emit the tst directly and fix the upper-case function names
later.